emission/core/get_database.py

```diff
@@ -15,9 +15,9 @@
 config = ecbc.get_config('conf/storage/db.conf',
     {"DB_HOST": "timeseries.url", "DB_RESULT_LIMIT": "timeseries.result_limit"})
 
-print("Retrieved config %s" % config)
 url = config.get("DB_HOST", "localhost")
 result_limit = config.get("DB_RESULT_LIMIT", 250000)
+print("Retrieved config: {'DB_HOST': '%s', 'DB_RESULT_LIMIT': '%s'}" % (url, result_limit))
```

Comment:
@MukuFlash03 the problem with printing these values **after** the `config.get` calls is that it is not sufficient for debugging. You can see the final value of the variables here, but you also see them on line 34. The point behind putting the "Retrieved config" before the `config.get` is so that we could see the inputs from the environment before they were modified by the defaults.

Reply:
Logging these values before using default values. 
Notes [in commit message](https://github.com/e-mission/e-mission-server/pull/975/commits/9bba6c8ef2187533caba7987f7c8a00c1fdd1c2d).